src/dune_rules/stanzas/copy_files.ml

```diff
@@ -7,6 +7,7 @@ type t =
   ; mode : Rule.Mode.t
   ; enabled_if : Blang.t
   ; files : String_with_vars.t
+  ; sources : bool
```

Comment:
maybe `files_from = Sources | Build` or something like that?

Reply:
Great idea. Something like https://github.com/ocaml/dune/pull/9827/commits/5802eae42adba5c909670a6d77dd3c331b1f1ead?

Reply:
yes!